python/cuml/dask/datasets/blobs.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from dask.dataframe import from_delayed
+import pandas as pd
+
+import cudf
+
+from dask.distributed import default_client
+from dask.distributed import wait as dask_wait
+
+from sklearn.datasets import make_blobs as skl_make_blobs
+
+import numpy as np
+
+import random
+import math
+
+
+def create_df(f, m, n, centers, cluster_std, random_state, dtype, r):
+    """
+    Returns Dask Dataframes on device for X and y.
+    """
+    X, y = skl_make_blobs(m, n, centers=centers, cluster_std=cluster_std,
```

Comment:
I think make_blobs on gpu is an example of something that would be super-easy to do in cupy when we have that fully integrated...

Reply:
cuml already has a make_blobs but it outputs row-major right now. The only reason I"m using skl's make_blobs in the meantime is because the size of the generated blobs is throwing a cuda exception when trying to transpose it using our tools in cuml. I think in a future PR we should offer an option in the C++ layer to generate blobs in column major format (I also think that is one of the reasons DBSCAN was failing when using it).